requirements/requirements.txt

```diff
@@ -1,3 +1,4 @@
+python>=3.6
```

Comment:
python is not a pip package
hence, it must not be here
we should specify this somewhere in the docs (or at least in readme) as I did in https://github.com/INM-6/viziphant-new/blob/master/doc/install.rst#prerequisites

Reply:
I reverted back to the previous commit.